admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/controller/ApplicationControllerTest.java

```diff
@@ -129,7 +129,7 @@ public void t6createApplicationFailTest() throws Exception {
 	@WithUserDetails("global-admin")
 	public void t1getAllApplicationFailTest() throws Exception {
 		MasterDataTest.checkResponse(mockMvc.perform(MockMvcRequestBuilders.get("/applicationtypes")).andReturn(),
-				"KER-MSD-002");
```

Comment:
why for failed test cases we are adding null or modify the test name

Reply:
Modified the name sir.